config/samples/ccruntime/peer-pods/peerpodsconfig.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: confidentialcontainers.org/v1alpha1
+kind: PeerPodConfig
+metadata:
+  name: coco-config
+  namespace: confidential-containers-system
+spec:
+  cloudSecretName: "peer-pods-secret"
+  configMapName:   "peer-pods-cm"
+  limit: "10"
```

Comment:
I think that default limit was changed to 1 in cloud-api-adaptor. Should we use the same here?